src/settings/FixedDueDateSchedule/FixedDueDateScheduleForm.js

```diff
@@ -130,6 +130,10 @@ class FixedDueDateScheduleForm extends React.Component {
     this.setState({ confirmDelete: true });
   };
 
+  scheduleList = () => {
```

Comment:
Could you please clarify where we use current function?

Reply:
Nowhere, just forget to clean up. Thanks!